std/permissions/mod.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+const { PermissionDenied } = Deno.errors;
+
+function getPermissionString(descriptors: Deno.PermissionDescriptor[]): string {
+  return descriptors
+    .map(pd => {
+      switch (pd.name) {
+        case "read":
+        case "write":
+          return pd.path ? `${pd.name}(${pd.path})` : pd.name;
+        case "net":
+          return pd.url ? `${pd.name}(${pd.url})` : pd.url;
+        default:
+          return pd.name;
```

Comment:
It could be really helpful for some people to stringify them to the CLI flags to get an error like
```
The following permissions have not been granted:
  --allow-env
  --allow-read=path/to/dir
```

Reply:
Yeah, that is better output...